trello_utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import requests
+
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TKN = os.environ.get('TRELLO_TKN')
+
+TRELLO_BOARD = '5f48fa1737fa77134ffe1e17'
+TRELLO_TODO_LIST = '5f48fa17b70abe80f7f0942d'
+TRELLO_DONE_LIST = '5f48fa17d5f15e744c86a725'
+TRELLO_URL_BASE = 'https://api.trello.com/1/'
+
+CARD_DONE_STATUS = 'Done!'
+CARD_TODO_STATUS = 'To-do'
+
+LOCAL_CARDS = []
+
+
+def trimCardsList(trello_cards):
```

Comment:
Why is this called "trimCardsList"? Looks to me like it's mapping the list of card dictionaries from Trello to a list of your Card class.